app/Exceptions/DockerNotAvailableException.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Exceptions;
+
+use App\Shell\Environment;
+use App\Shell\Shell;
+use Exception;
+
+class DockerNotAvailableException extends Exception
+{
+    public function render($request = null): void
+    {
+        $console = app('console');
+        $shell = app(Shell::class);
+        $environment = app(Environment::class);
+
+        $console->line('');
+        $console->line($shell->formatErrorMessage('Docker is not available.'));
+        $console->line('');
+
+        if ($environment->isLinuxOs()) {
+            $console->line('Verify that the Docker service is running:');
```

Comment:
Could we move Linux, Windows, and Mac into their own protected methods? That'll start to make this a bit more manageable.